hack/mock-based-tests.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
+
+openshift admin new-project mock --description="This is an example project to demonstrate OpenShift v3" --admin="e2e-user"
```

Comment:
our tests are self-contained, they don't assume a running/configured openshift instance..  can you update this to start (and terminate) its own instance?